statement.go

```diff
@@ -157,6 +157,16 @@ func (c converter) ConvertValue(v interface{}) (driver.Value, error) {
 		return int64(u64), nil
 	case reflect.Float32, reflect.Float64:
 		return rv.Float(), nil
+	case reflect.Bool:
```

Comment:
The changes in this file affect all versions (not just 1.8). Can you explain what this does? In particular what does this do to 1.7 and lower?

Reply:
Oh nvm I just saw that this is from https://github.com/golang/go/commit/d7c0de98a96893e5608358f7578c85be7ba12b25